### PR TITLE
Added data for trier city

### PR DESCRIPTION
**Pull request by operator2010**
added market data for trier city. Greetings from www.LokaleWare.de

### CODE REVIEW

**Comment by torfsen** [comment]
Hi @operator2010, thanks for adding data! There are still some problems with your pull request, as you can see the automatic check on Travis CI failed. If you click on [Details](https://travis-ci.org/wo-ist-markt/wo-ist-markt.github.io/builds/197440579) you will get the full log file. It includes the following part:

```
===> Validating cities/trier.json ...

undefined:53

  ],

  ^

SyntaxError: Unexpected token ]
```
So the `trier.json` file that you added has syntax errors. You can use for example an [online JSON editor](http://www.jsoneditoronline.org/) to figure out the problem and then update the pull request.

Feel free to ask again if you need more help.

File: cities/trier.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        "opening_hours": "Mar-Okt Sa 09:30-16:00"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          6.64295,
+          49.75651
+        ]
+      }
+    },
```
Comment (johnjohndoe) [inline cities/trier.json]: The error is caused by a `,` after the last `feature` before the `metadata` block. As pointed out by @torfsen you can easily spot the error if you paste the JSON content in the [JSON Editor Online tool](http://www.jsoneditoronline.org).

File: cities/cities.json [hunk 2]
```diff
@@ -87,10 +87,6 @@
         "id": "hilden",
         "label": "Hilden"
     },
-    "kaiserslautern": {
```
Comment (johnjohndoe) [inline cities/cities.json]: Further, please check the other files (_cities.json_, _README.md_): in your current commit you delete entries for Kaiserslautern and Zwickau. Please undo the deletion!
In _cities.json_ the entry for Trier is missing.

Comment (johnjohndoe) [inline cities/cities.json]: Please undo all changes to the entries of other cites in all files. Take a look at the "Files changed" tab here on this website - it shows you the "sum" of all commits. Please do not change the text of Zwickau. You can also preview the changes in the README.md if you just visit your `master` branch [here](https://github.com/operator2010/wo-ist-markt.github.io).

**Comment by johnjohndoe** [comment]
@operator2010 Please let me know if you need help with the change.

**Comment by johnjohndoe** [comment]
Hi @operator2010. It would be great if we could resolve the pending changes together to get your contribution merged.

**Comment by johnjohndoe** [comment]
@operator2010 Since I could no longer reach you I picked your contribution for Trier and applied the pending changes myself. I finally rebased and merged the resulting branch here: 20278b18bfe2670c4632574848e7ff13483867b4.

Thank you for add Trier to the project! In case you read that I like to notify you that I appreciate your effort. I like to point you to these two follow-up tasks:
- [ ] Send a tweet `#woistmarkt` with a **screenshot** and a **link** to the website (please share the tweet link here)
- [ ] Try to convince the city of Trier to **link** to our website, see #38

**Pull request closed without merging**